packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -76,17 +77,13 @@ export class SummaryManager extends EventEmitter implements IDisposable {
         return this._disposed;
     }
 
-    /** Used to calculate number of ops since last summary ack for the current elected client */
-    private lastSummaryAckSeqForClient = 0;
-    private hasSummarizersInQuorum: boolean;
-    private hasLoggedTelemetry = false;
-
     constructor(
         private readonly context: IContainerContext,
-        private readonly summaryCollection: SummaryCollection,
+        summaryCollection: SummaryCollection,
+        clientElection: OrderedClientElection,
```

Comment:
>OrderedClientElection [](http://example.com/codeflow?start=24&length=21)

can we just take an SummarizerClientElection? it will require reworking how refresh summarizer is past, but i think i'd prefer the to be an event any way

Reply:
Okay done, but note that the event name will change with the follow-up PR.